api/src/main/java/edu/cornell/mannlib/vitro/webapp/audit/AuditResults.java

```diff
@@ -0,0 +1,39 @@
+/* $This file is distributed under the terms of the license in LICENSE$ */
+
+package edu.cornell.mannlib.vitro.webapp.audit;
+
+import java.util.List;
+
+/**
+ * Results object for retrieving entries from the audit store
+ */
+public class AuditResults {
+    private long total;
```

Comment:
Please make these final as they are set in the constructor.

Reply:
fixed